Todoey/Todoey/Data Model/Category.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Category.swift
+//  Todoey
+//
+//  Created by 中原護 on 2023/05/08.
+//  Copyright © 2023 App Brewery. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class Category: Object {
+    @objc dynamic var name: String = ""
```

Comment:
型推論が使える場合は、明示的に型を書かないことが多いです。

```suggestion
    @objc dynamic var name = ""
```